src/pem_common_plug.c

```diff
@@ -49,8 +51,13 @@ int pem_valid(char *ciphertext, struct fmt_main *self)
 	keeptr = ctcopy;
 
 	ctcopy += TAG_LENGTH;
-	if ((p = strtokm(ctcopy, "$")) == NULL) // type
+	p = strtokm(ctcopy, "$"); // type
+	if (p == NULL)
```

Comment:
Please don't make unnecessary and inconsistent style changes. Note how the rest of the function still uses the `if ((p = strtokm(NULL, "$")) == NULL)` way of writing these things (and this is also seen in many other source files), so your change makes it inconsistent.

Reply:
Ack